cfpb/cfgov-refresh
# Update BAH email signup blurbs for new boot camp

In conjunction with rolling out a new email-based homebuying boot camp on Wednesday, we need to update the two places where Buying a House email signup blurbs are hardcoded.

## Changes

- Updates hardcoded BAH email signup blurbs

## Testing

1. Pull branch
1. Load up http://localhost:8000/owning-a-home/ and trigger the popup (clearing local storage, if needed) to see the new language 
1. Load up http://localhost:8000/owning-a-home/explore-rates/ and scroll to the bottom to see the new language

## Checklist


## Testing checklist

### Browsers
